src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,71 @@
+package duke;
+
+import duke.commands.Command;
+import duke.tasktypes.TaskList;
+import java.util.Scanner;
+
+
+/**
+ * Represents the chat bot.
+ */
+public class Duke {
+
+    private Storage storage;
+    private TaskList taskList;
+    private Ui ui;
+
+    /**
+     * Constructor for the class.
+     * @param filePath filePath for the data file.
```

Comment:
Maybe there should be an empty line between description and parameter section? I think thats what they mean [here](https://se-education.org/guides/conventions/java/intermediate.html#comments)

I noticed this in the other javadocs too, so remember to fix those too if this comment makes sense.

```suggestion
     * Constructor for the class.
     *
     * @param filePath filePath for the data file.
```